plugins/plugin-testing-java/plugin-testing-junit/che-plugin-testing-junit-server/pom.xml

```diff
@@ -57,5 +61,23 @@
             <groupId>org.eclipse.che.plugin</groupId>
             <artifactId>che-plugin-testing-classpath-server</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.javassist</groupId>
+            <artifactId>javassist</artifactId>
+        </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.mycila</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>**/**/AbstractTestListener.java</exclude>
```

Comment:
In the new files I created, I added the copyright in the name of *RedHat, Inc*, and the license-checking maven plugin doesn't like that.